docs/content/v2.20/drivers-orms/nodejs/yugabyte-node-driver.md

```diff
@@ -75,9 +75,11 @@ The following table describes the connection parameters required to connect, inc
 | database | Database name | yugabyte |
 | user | Database user | yugabyte |
 | password | User password | yugabyte |
-| loadBalance | Enables [uniform load balancing](../../smart-drivers/#cluster-aware-load-balancing) | false |
-| topologyKeys | Enables [topology-aware load balancing](../../smart-drivers/#topology-aware-load-balancing). Specify comma-separated geo-locations in the form `cloud.region.zone:priority`. Ignored if `loadBalance` is false | Empty |
+| loadBalance | Enables [uniform load balancing](../../smart-drivers/#cluster-aware-load-balancing) | false(Disabled) |
+| topologyKeys | Enables [Topology-aware load balancing](../../smart-drivers/#topology-aware-load-balancing). It can be set to comma-separated geo-locations in the form `cloud.region.zone:priority`. Ignored if `load_balance` is false | Empty |
```

Comment:
```suggestion
| topologyKeys | Enables [topology-aware load balancing](../../smart-drivers/#topology-aware-load-balancing). It can be set to comma-separated geo-locations in the form `cloud.region.zone:priority`. Ignored if `loadBalance` is false. | Empty |
```